docs/static/breaking-changes.asciidoc

```diff
@@ -15,7 +17,9 @@ after installing it via a package (RPM, DEB).
 
 * **Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries.
 Previously it used to install in `/opt/logstash` directory. This change was done to make the user experience
-consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
+consistent with other Elastic products. Full directory layout is described in <<dir-layout>>. The source of release packages 
+have changed from `packages.elastic.co` to `artifacts.elastic.co`. For example, 5.x and all the patch releases in this series 
+source will be `https://artifacts.elastic.co/packages/5.x/apt`
```

Comment:
"series source" doesn't sound right to me. Would it be correct to say, "For example, 5.x and all the patch releases in this series will available at....